ENIGMAsystem/SHELL/Platforms/SDL/Joystick.cpp

```diff
@@ -22,11 +22,27 @@
 
 namespace {
 
-SDL_Joystick *joystick1 = NULL;
-SDL_Joystick *joystick2 = NULL;
+SDL_Joystick *joysticks[2] = { NULL, NULL };
```

Comment:
I know gm only supports 2 joysticks but we can support more by using std::vector<SDL_Joystick*> = {nullptr, nullptr} instead of hard coded array